README.md

```diff
@@ -2,478 +2,35 @@
-* [Git your reset on](https://changelog.com/podcast/480) - Podcast with the author of the above article.
-* [Pro Git Book](https://git-scm.com/book/en/v2) - The main resource for learning Git.
-* [Learn Git Branching](https://learngitbranching.js.org/) - Interactive browser game that helps you to learn Git branching.
+## Contents ✨
 
-Versioning:
-* [SemVer](https://semver.org) - Versioning scheme used by many.
-* [CalVer](https://calver.org/) - Another versioning scheme used by many.
-* [zero0ver](https://0ver.org/) - Parody website on how many libraries misuse Semver.
-* [Giving Your Firmware Build a Version](https://embeddedartistry.com/blog/2016/12/21/giving-your-firmware-build-a-version/) - Good article from Embedded Artistry about versioning in firmware related context.
-* [Proper Release Versioning Goes a Long Way](https://interrupt.memfault.com/blog/release-versioning) - Another good article about versioning from Memfualt.
+* [GitHub projects guidelines] - Describes general guidelines that are used across GitHub projects.
+* [Developer guidelines] - Describes guidelines specific to developers that should be followed when working on software/firmware projects.
+* [Clang Format] - Document dedicated to setting up the `clang-format` tool.
 
-Release naming:
-* [Release naming conventions](https://www.drupal.org/docs/develop/git/git-for-drupal-project-maintainers/release-naming-conventions) - Naming conventions used by Drupal.
+## Other repositores 🌍
 
-Changelog:
-* [What makes a good changelog?](https://depfu.com/blog/what-makes-a-good-changelog) - Writing a good changelog, like writing any text, is about knowing your audience and their needs.
+* [Github Action workflows] - Contains a variety of GitHub Actions workflow files that automate various processes.
+* [Generic project template] - Used for creating a new GitHub project, 🚧 **currently under construction** 🚧.
+* [Zephyr project template] - Used for creating a new Zephyr project, 🚧 **currently private and under construction** 🚧.
 
-Awesome projects:
-* [Awesome embedded](https://github.com/nhivp/Awesome-Embedded) - A curated list of awesome embedded resources.
-* [Awesome C](https://github.com/oz123/awesome-c) - A curated list of C good stuff.
-* [Awesome Zephyr RTOS](https://github.com/golioth/awesome-zephyr-rtos) - A curated list of Zephyr related stuff.
 
-[GitFlow]: https://nvie.com/posts/a-successful-git-branching-model
-[SemVer]: https://semver.org
-[Keep a Changelog's]: https://keepachangelog.com/en/1.0.0/
+[GitHub projects guidelines]: docs/github_projects_guidelines.md
+[Developer guidelines]: docs/developer_guidelines.md
+[Clang Format]: clang-format/README.md
+[Github Action workflows]: https://github.com/IRNAS/irnas-workflows-software
+[Generic project template]: https://github.com/IRNAS/irnas-projects-template
+[Zephyr project template] - Used for creating a new Zephyr project, 🚧 **currently under construction** 🚧.
```

Comment:
`:` instead of `-`?
![image](https://user-images.githubusercontent.com/26332685/168799010-ad55e289-7cfc-43bb-b4f7-3157740c75f8.png)

Reply:
This should not be here, I will remove it.